File: src/tests/Markify.Fixtures/Projects/Source/Class/InheritedClass.cs
using System;
using System.Collections.Generic;

public class InheritClass : Exception{}

public class ImplementInterfaceClass : IDisposable{}

public class ImplementGenInterfaceClass : IList<String>, IReadOnlyCollection<String>{}

public class MixedInheritanceClass : Exception, IList<String>, IDisposable{}

File: src/tests/Markify.Fixtures/Projects/Source/Class/InheritedClass.vb
Imports System
Imports System.Collections.Generic

Namespace FooSpace.InnerSpace
    Public Class InheritClass
        Inherits Exception
    End Class
End Namespace

Public Class ImplementInterfaceClass
    Inherits IDisposable
End Class

Public Class ImplementGenInterfaceClass
    Inherits IList(Of String), IReadOnlyCollection(Of String)
End Class

Public Class MixedInheritanceClass
    Inherits Exception, IList(Of String), IDisposable
End Class